app/Services/ObjectManager/EloquentObjectManager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\ObjectManager;
+
+use Illuminate\Database\Eloquent\Model;
+
+class EloquentObjectManager implements ObjectManagerInterface {
```

Comment:
Misschien hier even een docstring aan toevoegen om uit te leggen waarom deze class nodig is